index.html

```diff
@@ -2643,18 +2727,18 @@ <h3>Events</h3>
         </p>
         <section id="http-sse-profile-protocol-binding-subscribeevent">
           <h5><code>subscribeevent</code></h5>
-          <p class="rfc2119-assertion" 
+	  <span class="rfc2119-assertion" 
             id="http-baseline-profile-protocol-binding-subscribeevent-1">
```

Comment:
This id should be be `http-sse-profile-protocol-binding-subscribeevent-1`. That's not the fault of this PR, but probably easier to fix in this PR rather than create another merge conflict.

Reply:
sure, changed